### PR TITLE
Updated copyright info for README

### LINKED ISSUE #1251: Update copyright information on README
Per feedback from legal: Update copyright info on README with: 

>>> go-ipfs:

The go-ipfs project is dual-licensed under Apache 2.0 and MIT terms, as explained in the [Permissive License Stack](https://protocol.ai/blog/announcing-the-permissive-license-stack/):

Apache License, Version 2.0, ([LICENSE-APACHE](https://github.com/ipfs/go-ipfs/blob/master/LICENSE-APACHE) or http://www.apache.org/licenses/LICENSE-2.0)
MIT license ([LICENSE-MIT](https://github.com/ipfs/go-ipfs/blob/master/LICENSE-MIT) or http://opensource.org/licenses/MIT)

### PR DESCRIPTION
**Pull request by dawnkelly09**
Per feedback from legal Fixes #1251

### CODE REVIEW

**Comment by dawnkelly09** [comment]
This is waiting on final clarification from Legal team. Do not merge yet! Thanks!

**Comment by johnnymatthews** [comment]
Converted this PR to draft, just until it's ready for review.

File: readme.md
Comment (johnnymatthews) [inline readme.md:22]:
```suggestion
```

File: readme.md
Comment (johnnymatthews) [inline readme.md:66]:
```suggestion
```

File: readme.md
Comment (johnnymatthews) [inline readme.md:2]:
```suggestion
# Filecoin Docs
```

File: readme.md
Comment (johnnymatthews) [inline readme.md:1]:
```suggestion
# Filecoin Docs

```

**Comment by mishmosh** [comment]
I think this PR is now outdated (since it adds the README.md file afresh, which was completed in #1249. Should merge `main` into this branch?

**Comment by johnnymatthews** [comment]
Just merged `main` into this branch, so everything's all caught up now.

**Comment by mishmosh** [comment]
@dawnkelly09 It's because you created this branch from `main` before #1249 was merged, so to github it looks like you're effectively PRing the entire new readme.md file twice. 

More info here: https://docs.gpcrdb.org/git_workflow.html

**Comment by dawnkelly09** [comment]
Please just confirm if we are all happy with this so it can be merged. Or if it is better to just close it at this point. Thank you!

File: readme.md
Comment (mishmosh) [reply readme.md:62]: This repeats info already contained in line 60. `LICENSE-APACHE` would usually link to a new file containing a copy of the license, but for simplicity's sake let's not create new license files solely for the docs project, we can link to the published versions.

**Review by mishmosh — CHANGES_REQUESTED** [review]
Suggestions made for lines 60-63.

**Pull request merged**
